bigtop_toolchain/manifests/packages.pp

```diff
@@ -293,21 +293,9 @@
         "python-setuptools",
         "libffi-dev",
         "python3-dev",
-        "python2.7-dev",
         "nasm",
         "yasm"
       ]
-      if (($operatingsystem == 'Ubuntu' and 0 <= versioncmp($operatingsystemmajrelease, '22.04'))) {
-        file { '/usr/bin/python':
-          ensure => 'link',
-          target => '/usr/bin/python2',
-        }
-        $pkgs = $_pkgs
-      } elsif (($operatingsystem == 'Ubuntu' and 0 <= versioncmp($operatingsystemmajrelease, '20.04')) or ($operatingsystem == 'Debian' and 0 <= versioncmp($operatingsystemmajrelease, '11'))) {
-        $pkgs = concat($_pkgs, ["python-is-python2"])
-      } else {
-        $pkgs = $_pkgs
```

Comment:
This breaks build on Ubuntu and Debian due to undefined `pkgs`. @JiaLiangC.

```
Warning: Unknown variable: 'pkgs'. (file: /home/admin/srcs/bigtop/bigtop_toolchain/manifests/packages.pp, line: 305, column: 13)
Error: Evaluation Error: Missing title. The title expression resulted in undef (file: /home/admin/srcs/bigtop/bigtop_toolchain/manifests/packages.pp, line: 305, column: 13) on node ip-172-31-163-160.ap-northeast-1.compute.internal

> Task :toolchain FAILED

FAILURE: Build failed with an exception.
```

Reply:
@iwasakims  fixed
![image](https://github.com/apache/bigtop/assets/18082602/5ea1dbab-a1b0-40b0-b809-3cefb1463fe1)